packages/flow/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll": true
+  },
+  "cSpell.words": ["Astro"],
+  "prettier.documentSelectors": ["**/*.astro"]
+}
```

Comment:
This seems highly opinionated. Is this a recommended pattern for all Astro integrations or components?

Reply:
I used the official typescript community template for this project. I did not change anything about it. I'm willing to remove that stuff